zookeeper-api/src/main/java/org/apache/helix/zookeeper/zkclient/callback/ZkAsyncCallbacks.java

```diff
@@ -198,12 +211,14 @@ public void callback(int rc, String path, Object ctx) {
       }
 
       _rc = rc;
+      _path = path;
```

Comment:
Why we need to update this path?

Reply:
I added the _path car so that it can be acquired by caller when implementing async for metaclient.